RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -594,14 +586,14 @@ namespace SDL {
     float multDzDr = dzOutInAbs * coshEta / (coshEta * coshEta - 1.f);
 
     float kZ = (zOut - zIn) / dzSDIn;
-    float sdlThetaMulsF = 0.015f * alpaka::math::sqrt(acc, 0.1f + 0.2f * (rtOut - rtIn) / 50.f);
```

Comment:
`0.015f` is repeated quite often and apparently now goes squared; perhaps define kMulsInGev2 (the constant is the 15 MeV from the approximate Bethe-Bloch formula)